docs/server/installation.rst

```diff
@@ -124,10 +79,7 @@ To activate the virtual environment run the :command:`activate` script:
 
   $ source env/bin/activate
```

Comment:
Yes, but I tend to think on narrowing the possibilities. If you provide a dead simple path, users will use it (mostly), and when it comes to debug reported issues it helps that people is using something closer to what you use.

But it is just a personal preference. We can decide not to add it, or add it in the future.